src/doc/src/reference/build-scripts.md

```diff
@@ -288,6 +292,7 @@ if foo_bar_condition {
 }
```

Comment:
Unrelated to this PR. Instead of putting an example here, do you think it's better to just have a link to [the conditional compilation build script example](https://doc.rust-lang.org/nightly/cargo/reference/build-script-examples.html#conditional-compilation)?

Reply:
I think having a example right in the documentation is very valuable, however linking to a more complete example is quite valuable to, so I added a link to it.